packaging/nominatim-api/pyproject.toml

```diff
@@ -19,7 +19,10 @@ dependencies = [
     "pyYAML>=5.1",
     "SQLAlchemy>=1.4.31",
     "psycopg",
-    "PyICU"
+    "PyICU",
+    "cantonese-romanisation",
+    "unidecode",
+    "opencc"
```

Comment:
These libraries should preferably be [optional dependencies](https://hatch.pypa.io/latest/config/metadata/#optional).